tendrl/gluster_integration/persistence/persister.py

```diff
@@ -99,17 +37,72 @@ def update_tendrl_context(self, context):
     def update_tendrl_definitions(self, definition):
         self._store.save(definition)
 
-    def _run(self):
-        LOG.info("Persister listening")
 
-        while not self._complete.is_set():
-            gevent.sleep(0.1)
-            pass
+class GlusterIntegrationFilePersister(FilePersister):
```

Comment:
As we discussed we don't need to implement this as part refactoring We will take this up whenever required